build.gradle.kts

```diff
@@ -38,16 +33,13 @@ allprojects {
         mavenLocal()
         google()
         mavenCentral()
-        jcenter {
-          content {
-            includeModule("com.facebook.yoga", "proguard-annotations")
-            includeModule("com.facebook.fresco", "stetho")
-          }
+        maven {
```

Comment:
Hm, I believe we also don't need jitpack here
Only FB specific ones were remaining, according to the list above, and they should be available on Maven Central

Reply:
Yes, I can remove it but tomorrow